syntax = "proto3";

package rxsamplesmetadata;

message RxSamplesMetadata {
  uint32 sequence_num = 1;
  uint32 numberofreceivesamples = 2;
  double rx_rate = 3;
  double sequence_time = 4;
  double initialization_time = 5;
  double sequence_start_time = 6;
  uint64 ringbuffer_size = 7;
  bool agc_high = 8;
  bool lp_high = 9;
}
